disabled_pylint_errors.txt

```diff
@@ -0,0 +1,15 @@
+### ABOUT PYLINT CONFIG ###
```

Comment:
Is this file also needed along with the pylintrc file? If so, where do we place this file? If it also belongs to the Python27 directory, please also share this on Google Drive instead.